GDJS/tests/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "private": true,
   "scripts": {
-    "test": "./node_modules/karma/bin/karma start --browsers ChromeHeadless --single-run",
+    "test": "cd ./node_modules/karma/bin && karma start ../../../karma.conf.js --browsers ChromeHeadless --single-run",
```

Comment:
@4ian this was pushed by mistake 😬, but still, can you check whether this works correctly on macOS/linux or not? If yes, I guess we can keep it?

Reply:
It works :) 
Can you try though just: `karma start --browsers ChromeHeadless --single-run`?
(and `karma start --browsers Firefox --single-run`?)

If it works, let's use this. In theory, npm is responsible for going into the `node_modules` folder and running the appropriate program when you just write the program name like "karma".

So `karma start --browsers ChromeHeadless --single-run` should work. I don't know why I put originally `./node_modules/karma/bin/karma`, I guess I was not aware that it would work with just `karma`.